cmd/gateway/dot/dot.go

```diff
@@ -68,15 +69,16 @@ func init() {
 	configCmd.AddCommand(app.ConfigApplyCmd.Cmd)
 
 	// Extra flag additions for gateway_dot -----------------------------------------------
+	keystorePath, keystorePassPath, _ := keystore.GetKeystoreDetails()
```

Comment:
No test of `error`. May break intended CLI functionality later. Either check for error or comment above that in case of error, `keystorePath` and `keystorePassPath` assumes certain values (which here is "").